first-analysis-steps/files-from-grid.md

```diff
@@ -94,7 +94,7 @@ if __name__ == '__main__':
     files = FILES[:n_files]
     for f in files:
         print('Getting file {0}.'.format(f))
-        call('dirac-dms-get-file {0}'.format(f), shell=True)
+        call('lb-dirac dirac-dms-get-file LFN:{0}'.format(f), shell=True)
```

Comment:
Hi, Martha! Thanks for looking into the lesson! I think if you do this, then the following `lb-dirac python getEntries.py [n]` command will not work. Also `LFN:` or no `LFN:` would depend on the names you provide in the script. I would keep without `LFN:`, since the `.DST.py` file that students download during the previous bkk lesson has string with `LFN:` as part of the name. 

The normal code works for me now.